app/jobs/appointment_notification/missed_visit_job.rb

```diff
@@ -1,18 +1,19 @@
 class AppointmentNotification::MissedVisitJob
-  include Notifiable
   include Sidekiq::Worker
 
   sidekiq_options queue: "high"
 
-  def perform(schedule_hour_start, schedule_hour_finish)
+  def perform
```

Comment:
This job no longer accepts the time window, since the `AppointmentNotificationService` invoked below can find that out itself.